bors.toml

```diff
@@ -1 +1,10 @@
-status = ["continuous-integration/travis-ci/push"]
+status = [
+    # Travis CI
+    "continuous-integration/travis-ci/push",
+    # GitHub Actions
+    "Build and test on ubuntu-latest",
+    "Build and test on windows-latest",
+    "Build and test on macOS-latest",
+    "Checking fmt and docs",
+    "Clippy check",
```

Comment:
Unfortunately, as far as I know, all job names need to be written. I have checked that the pattern mentioned in [documentation](https://bors.tech/documentation/) cannot handle this properly (https://github.com/taiki-e/derive_utils/pull/22).
> Patterns are written using SQL LIKE syntax. For example, this status directive will match any CI job that starts with ci/gitlab/:

Reply:
ah dang yeah that's unfortunate but okay I think. Thanks for checking this!

Reply:
Hm actually we may want to fill in the full matrix here. I think keeping the version as part of the name may be better and set us up for beta testing also (which we'll probably want to start doing soon, in addition to still testing nightly!)

Reply:
Ah nevermind, we can cross that bridge once we get to it. Thanks!